clients/algoliasearch-client-javascript/packages/client-common/tsconfig.json

```
@@ -2,8 +2,9 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "types": ["node", "vitest/globals"],
-    "outDir": "dist"
+    "outDir": "dist",
+    "skipLibCheck": true
```

```suggestion
    "outDir": "dist"
```

does it still work without it?

---

nope, still complains about rollup

---

(╯°□°)╯︵ dnʅʅoɹ